docs/installation/linux.md

```diff
@@ -39,7 +39,7 @@ This is the preferred method.
 curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
 
 # Register the Microsoft Ubuntu repository
-curl https://packages.microsoft.com/config/ubuntu/14.04/prod.list | sudo tee /etc/apt/sources.list.d/microsoft.list
+sudo curl -o /etc/apt/sources.list.d/microsoft.list https://packages.microsoft.com/config/ubuntu/14.04/prod.list
```

Comment:
Should we remove 14.04 - the version is out of support?

Reply:
I had a hosting provider that will go unnamed that _just_ upgraded a VPS from 14.04.  It's EOL from Canonical, but there's definitely some users out there still on it. :p